src/test/scala/chiselTests/AutoClonetypeSpec.scala

```diff
@@ -85,10 +84,8 @@ class RecordAutoCloneType[T <: Data](gen: T) extends Record with AutoCloneType {
 // NOTE: This is a very weird thing to do, don't do it.
 class RecordWithVerbotenMethods(w: Int) extends Record {
```

Comment:
does this test still make any sense?

Reply:
nice catch, yeah i don't think so

Reply:
It still does `_usingPlugin` and `_elementsImpl` which are checked for Bundles but not Records so it's exercising that code path.